### PR TITLE
docs: add Playwright preference note

### PR DESCRIPTION
**Pull request by stefanjudis**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

Follow [this slack conversation](https://checklyhq.slack.com/archives/C024ZAJ938S/p1657890513244319) we want to make sure to give info on our Playwright preference in the documentation. 

I also shuffled things around a bit, to streamline the order of information. E.g. explain what a browser check is before showing some code.

[Here's the preview](https://checklyhq-oxnxswywc-checkly.vercel.app/docs/browser-checks/).

### CODE REVIEW

File: site/content/docs/browser-checks/_index.md [hunk 1]
```diff
@@ -9,22 +9,43 @@ aliases:
...
 ---
 
+This quick start gives you all the info to create your first Browser check with Checkly. You should have some prior
+knowledge of working with Javascript and/or Node.js.
 
+## What is a Browser check?
 
-This quick start should give you all the info to create your first browser check with Checkly. You should have some prior
-knowledge working with Javascript and/or Node.js.
+A Browser check is a Node.js script that controls a Chrome browser to mimic user behavior.
```
Comment (ragog) [inline site/content/docs/browser-checks/_index.md:17]: @stefanjudis I would say "...script that controls a headless Chromium browser..."

File: site/content/docs/browser-checks/_index.md [hunk 2]
```diff
@@ -9,22 +9,43 @@ aliases:
...
+Load a web page, click a link, fill a form input – do everything your visitors might do and check if these interactions lead to the correct results.
 
-## What is a browser check?
+Your critical interactions might be:
 
-The five lines of code below are already a valid browser check. Its usefulness might be not that great, but still...
+- That the shopping cart is visible.
```
Comment (ragog) [inline site/content/docs/browser-checks/_index.md:22]: The first point doesn't work. "That the shopping cart is visible" is not an interaction. Verifying that an element appears is more of an assertion. I would substitute this with a third interaction. We can add a one liner below explaining how assertions factor into the equation (automated interactions + assertions = confidence things are working as expected).

File: site/content/docs/browser-checks/_index.md [hunk 3]
```diff
@@ -9,22 +9,43 @@ aliases:
...
+- That users can add products to the shopping cart.
+- That users can log into my app.
+
+Use Browser checks to guarantee that your site works as expected.
+
+Checkly uses the **[Playwright](https://github.com/microsoft/playwright)** and **[Puppeteer](https://github.com/GoogleChrome/puppeteer)**
+frameworks to power your Browser checks. Use these frameworks to control the interactions you want to happen on a web page.
+
+{{< info >}}
+While Playwright and Puppeteer share many similarities, they have evolved at different speeds over the last two years. Playwright's rapid release cycle and new  features such as [auto-waiting](https://playwright.dev/docs/actionability) and [the built-in inspector](https://playwright.dev/docs/debug#playwright-inspector) made it gain momentum in the developer community.
```
Comment (ragog) [inline site/content/docs/browser-checks/_index.md:32]: I would be even bolder here and add "We strongly recommend using Playwright if you are just starting out."

Additional idea would be linking the migration script, too: "If you would like to migrate from Puppeteer to Playwright, you can use our [migration script](https://github.com/checkly/puppeteer-to-playwright)."

File: site/content/docs/browser-checks/_index.md [hunk 4]
```diff
@@ -33,23 +54,17 @@ await browser.close()
...
-- Is my shopping cart visible?
-- Can users add products to the shopping cart?
-- Can users log in to my app?
-
-Checkly uses the **[Playwright](https://github.com/microsoft/playwright)** and **[Puppeteer](https://github.com/GoogleChrome/puppeteer)**
-frameworks to drive these actions. Playwright and Puppeteer are Javascript frameworks that "talk" to a real Google Chrome browser.
-You use these frameworks to control the interactions you want to happen on a web page.
-
 {{< info >}}
 Checkly currently supports using only **Google Chrome** with both Playwright and Puppeteer.
```
Comment (ragog) [inline site/content/docs/browser-checks/_index.md:69]: If we change to Chromium above, we might want to change it here, too.

**Review by ragog — CHANGES_REQUESTED** [review]

File: site/content/docs/browser-checks/_index.md (hunk 1 above)
Comment (stefanjudis) [reply site/content/docs/browser-checks/_index.md:17]: 👍

File: site/content/docs/browser-checks/_index.md (hunk 3 above)
Comment (stefanjudis) [reply site/content/docs/browser-checks/_index.md:32]: Nice. I like that!

File: site/content/docs/browser-checks/_index.md (hunk 2 above)
Comment (stefanjudis) [reply site/content/docs/browser-checks/_index.md:22]: 👍

**Review by ragog — APPROVED** [review]
LGTM

**Pull request merged**